tools/pythonpkg/src/include/duckdb_python/pybind11/conversions/render_mode_enum.hpp

```diff
@@ -0,0 +1,66 @@
+#pragma once
+
+#include "duckdb/common/common.hpp"
+#include "duckdb/common/exception.hpp"
+#include "duckdb/common/string_util.hpp"
+#include "duckdb/common/box_renderer.hpp"
+
+using duckdb::InvalidInputException;
+using duckdb::RenderMode;
+using duckdb::string;
+using duckdb::StringUtil;
+
+namespace py = pybind11;
+
+static RenderMode RenderModeFromString(const string &type) {
+	auto ltype = StringUtil::Lower(type);
```

Comment:
Instead of adding this to the Python module could we add `: uint8_t` to `RenderMode` so that the string conversion methods are generated by `scripts/generate_enums.py`?